wisp/wisp-launchdarkly/src/main/kotlin/wisp/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -70,6 +71,9 @@ class LaunchDarklyFeatureFlags @JvmOverloads constructor(
             buildUser(feature, key, attributes),
         )
         checkDefaultNotUsed(feature, result)
+        GlobalTracer.get()?.activeSpan()?.let { span ->
```

Comment:
Is this something we can test?

Reply:
I am curious about this too, @qi-liao , do you know?

Reply:
no idea